Dockerfile

```diff
@@ -33,66 +33,40 @@
 #
 # =================================================================
 
-FROM alpine:3.11
+FROM python:3.8-slim-buster
 LABEL maintainer="massimods@met.no,aheimsbakk@met.no"
 
-ARG PYCSW_HOME=/tmp/pycsw
+RUN apt-get update && apt-get install --yes \
```

Comment:
Should we add `--no-install-recommends` here (recommended by https://www.fromlatest.io) ?